Create transparent-proxy.adoc (#282)
PaloAltoNetworks/prisma-cloud-docs: cns/saas/admin_guide/start/enforcer/enforcer.adoc

//'''

. Ensure that your target host or cluster meets the xref:reqs.adoc[system requirements].

> . If there is transparent proxy in the data path between enforcers and Prisma Cloud SaaS console you will need to enable support for xref:transparent-proxy.adoc.adoc[non-transparent proxy]

Reviewer: ```suggestion
. If there is a transparent proxy in the data path between your enforcers and the Prisma Cloud SaaS console, you must enable support for a xref:./transparent-proxy.adoc[non-transparent proxy].
```
